docs/data/pages.ts

```diff
@@ -236,6 +233,34 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/x/migration',
+    title: 'Migration',
+    icon: 'BookIcon',
+    children: [
+      {
+        pathname: '/x/migration',
+        subheader: 'Upgrade to v6',
+        children: [
+          { pathname: '/x/migration/migration-data-grid-v5', title: 'Breaking changes: Data Grid' },
+        ],
+      },
+      {
+        pathname: '/x/migration',
+        subheader: 'Earlier versions',
+        children: [
+          {
+            pathname: '/x/migration/migration-pickers-lab',
+            title: 'Migration from lab to v5 (pickers)',
```

Comment:
The correct naming would be "date & time pickers" but that's very long for the nav bar